app/components/containerInspect/containerInspectController.js

```diff
@@ -1,12 +1,12 @@
-angular.module('containerInspect', [])
+angular.module('containerInspect', ['angular-json-tree'])
 .controller('ContainerInspectController', ['$scope', '$transition$', 'Notifications', 'ContainerService',
 function ($scope, $transition$, Notifications, ContainerService) {
   function initView() {
```

Comment:
Please relocate the `$scope.state` declaration here.

Reply:
OK.